### DISCUSSION TITLE
Mongo shell not found in Container

### DISCUSSION THREAD

**Discussion by gthrn** [docker-library/mongo#563] [General]
Hi all,
I really hope someone here can enlighten me:

I am starting a container like so:
`docker run --name some-mongo -d mongo:latest`

I then enter the container bash
`docker exec -it some-mongo bash`

and then I run
`mongo`
which results in
`bash: mongo: command not found`

I do not know what I am doing wrong. I want to use the "mongo" command for a docker-compose healthcheck, so I really need it to work.

Any Idea?

**Comment by yosifkit** [comment]
https://www.mongodb.com/docs/manual/release-notes/6.0-compatibility/#legacy-mongo-shell-removed

- `mongo` was removed by MongoDB Inc in 6.0
- `mongosh` is available in 5+

Refs: #558 #560 #561

**Reply by gthrn** [reply]
well thank you. It was easy after all
